src/types.db

```diff
@@ -192,6 +192,27 @@ snr			value:GAUGE:0:U
 spam_check		value:GAUGE:0:U
 spam_score		value:GAUGE:U:U
 spl			value:GAUGE:U:U
+stenographer_aged_files value:COUNTER:0:U 
```

Comment:
Please use `DERIVE` instead of `COUNTER`: With 64bit counters overflows are so rare that counter-resets are a much larger problem. Especially when the maximum value is unset. When using `COUNTER`, you will get *huge* spikes every time that stenographer is restarted.